instrumentation/log4j/log4j-context-data/log4j-context-data-common/testing/src/main/resources/log4j2-test.xml

```diff
@@ -2,7 +2,7 @@
 <Configuration status="WARN" packages="com.example.appender">
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} traceId: %X{trace_id} spanId: %X{span_id} flags: %X{trace_flags} - %msg%n" />
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} traceId: %X{trace_id} spanId: %X{span_id} flags: %X{trace_flags} baggage.baggage_key: %X{baggage.baggage_key} - %msg%n" />
```

Comment:
confirmed that `baggage.baggage_key` is resolved correctly in the test console output